src/templates/main/500.html

```diff
@@ -1,5 +1,61 @@
-{% extends 'main/base.html' %}
-{% block title %}Server error{% endblock %}
-{% block content %}
-<h1>Something went wrong</h1>
-{% endblock %}
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Something went wrong</title>
+  <style>
```

Comment:
Although there are a lot of CSS tags in this template already and it would be good to have them in a separated static file, I think its ok to keep them here for now, since this template don't intend to grow by the time